### PR TITLE
As a development team we want to structure our component lib using atomic design pattern

### PR DESCRIPTION
**Pull request by lspk129**
https://tree.taiga.io/project/troublemaker-website/us/65

### CODE REVIEW

File: components/organisms/Navbar/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        bgColor={props.muiTheme.palette.accent1Color}
+        zIndex={2}
+      >
+        <Row center={'xs'}>
+          <Col>
+            <AnimatedBorder
+              color={props.muiTheme.palette.whiteText}
+              onClick={() => Scroll.animateScroll.scrollToTop()}
+            >
+              HOME
```
Comment (tmxyz) [inline components/organisms/Navbar/index.js]: Don't leave naked text. Wrap it inside Text component. Add a prop to make it uppercase

File: components/organisms/Service/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import Title from '../../atoms/Title';
+
+const Service = () => (
+  <Division bg src={'../../../static/Service_image.jpg'} pb={3}>
+    <Division py={3}>
+      <Title tag={'h2'}>What we do</Title>
+    </Division>
+    <Row center={'xs'}>
+      <Col md={3} sm={4} xs={12}>
+        <DivisionHover bgcolor height={260}>
```
Comment (tmxyz) [inline components/organisms/Service/index.js]: No height

File: components/atoms/AnimatedBorder/index.js [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
+const AnimatedBorder = styled.div`
+  ${({ color }) => color && `color: ${color}`};
   text-align: center;
   padding: 14px 16px;
   text-decoration: none;
   font-size: 16px;
   cursor: pointer;
-  display: inline-block;
   &:after {
-    display: block;
```
Comment (lspk129) [inline components/atoms/AnimatedBorder/index.js]: Fixed.

File: components/atoms/AnimatedBorder/index.test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import AnimatedBorder from './';
+
+describe('AnimatedBorder', () => {
+  it('renders correctly', () => {
+    const wrapper = renderer.create(<AnimatedBorder />);
```
Comment (lspk129) [inline components/atoms/AnimatedBorder/index.test.js]: Fixed.

File: components/atoms/Division/index.test.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('render with textAlign props', () => {
+    const wrapper = renderer.create(<Division textAlign={'center'} />);
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
+
+  it('render with margin props', () => {
+    const wrapper = renderer.create(<Division m />);
```
Comment (lspk129) [inline components/atoms/Division/index.test.js]: Fixed.

File: components/atoms/ColStyled/index.test.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+describe('ColStyled', () => {
+  it('renders correctly', () => {
+    const wrapper = renderer.create(<ColStyled />);
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
+
+  it('renders with color props', () => {
+    const wrapper = renderer.create(<ColStyled color={'red'} />);
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
```
Comment (lspk129) [inline components/atoms/ColStyled/index.test.js]: Fixed.

File: components/atoms/DivisionHover/index.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  ${({ height }) => height && `height: ${height}px`};
+  ${({ scale }) => scale && `
+    img {
+      width: 200px;
+      height: 200px;
+    }
+    &:hover {
+      transform: scale(1.1);
+  }
+
```
Comment (lspk129) [inline components/atoms/DivisionHover/index.js]: Fixed.

File: components/atoms/DivisionHover/index.test.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders height props', () => {
+    const wrapper = renderer.create(<DivisionHover height={100} />);
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
+
+  it('renders scale boolean props', () => {
+    const wrapper = renderer.create(<DivisionHover scale />);
```
Comment (lspk129) [inline components/atoms/DivisionHover/index.test.js]: Fixed.

File: components/atoms/DivisionHover/index.test.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
+
+  it('renders background color boolean props with theme props', () => {
+    const wrapper = renderer.create(<DivisionHover bgcolor />);
```
Comment (lspk129) [inline components/atoms/DivisionHover/index.test.js]: Fixed.

File: components/atoms/Image/index.js [hunk 10]
```diff
@@ -1,8 +1,7 @@
 import styled from 'styled-components';
 
 const Image = styled.img`
-  width: 100px;
-  padding-top: 10px;
+  ${({ width }) => width && `width: ${width}rem`}
```
Comment (lspk129) [inline components/atoms/Image/index.js]: Fixed.

File: components/atoms/Title/index.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```
Comment (lspk129) [inline components/atoms/Title/index.js]: Fixed.

File: components/atoms/Title/index.test.js [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+import Title from './';
+
+describe('Title', () => {
+  it('renders correctly', () => {
+    const wrapper = renderer.create(<Title />);
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
+
+  it('renders with tag props', () => {
+    const wrapper = renderer.create(<Title tag={'h1'} />);
```
Comment (lspk129) [inline components/atoms/Title/index.test.js]: Fixed.

File: components/organisms/Navbar/index.js (hunk 1 above)
Comment (lspk129) [inline components/organisms/Navbar/index.js]: Fixed.

File: components/organisms/Service/index.js (hunk 2 above)
Comment (lspk129) [inline components/organisms/Service/index.js]: Fixed.

**Pull request merged**